rust-code-analysis-web/Cargo.toml

```diff
@@ -8,9 +8,6 @@ keywords = ["metrics"]
 description = "Run a web service to compute and export code metrics"
 license = "MPL-2.0"
 
-[[bin]]
-name = "rust-code-analysis-web"
```

Comment:
Will the release still contain a "web" binary if we do this?

Reply:
Yes, because the name of the binary is the same filename of the Rust file contained in the `bin` directory